examples/react/full-width-resizable-table/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "tanstack-table-example-full-width-resizable-table",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "start": "vite"
+  },
+  "dependencies": {
+    "@faker-js/faker": "6.3.1",
+    "@tanstack/react-table": "8.0.0",
+    "faker": "^6.6.6",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
+  },
+  "devDependencies": {
+    "@rollup/plugin-replace": "^4.0.0",
```

Comment:
add faker type as dev dependency.
https://github.com/TanStack/table/blob/main/examples/react/column-ordering/package.json#L19

Reply:
@types/faker has been added to devDependencies